Reorder README text

Shreeshrii opened the pull request:
Add an Issue Template
Reorganize Readme.MD

egorpugin commented on README.md:
```diff
@@ -1,16 +1,9 @@
-    
-https://github.com/tesseract-ocr/tesseract/blob/master/README.md
-  
-### Build 
```
I want this line back.

stweil replied:
It was moved to the end of the file.

egorpugin commented on README.md:
```diff
@@ -1,16 +1,9 @@
-[![Build Status](https://travis-ci.org/tesseract-ocr/tesseract.svg?branch=master)](https://travis-ci.org/tesseract-ocr/tesseract)
-[![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819/branch/master?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
-
-### Other
```
too

stweil replied:
They are still there, but with a better text.

egorpugin replied:
Where? The idea behind this caption was to split badges.
In this PR they are again near each other.

stweil commented:
The new README.md can be seen [here](https://github.com/Shreeshrii/tesseract).

stweil commented on README.md:
```diff
@@ -20,48 +13,28 @@ The lead developer is Ray Smith. The maintainer is Zdenko Podobny.
-[Release Notes](https://github.com/tesseract-ocr/tesseract/wiki/ReleaseNotes)
-
-# For developers
+The latest stable version is **[3.05.00](https://github.com/tesseract-ocr/tesseract/releases/tag/3.05.00)**, released in February 2017. Source code is available from [3.05 branch on github](https://github.com/tesseract-ocr/tesseract/tree/3.05). 3.05.01 bug-fix release is expected in May/June 2017.
```
s/on github/on GitHub/

Shreeshrii commented:
Thanks for your feedback.

I was looking at the Readme from the point of view of someone who is
looking at it on github.com and thought that the latest Readme.md would be
visible to them there, hence moved it to the bottom.

This text as well as the badges seemed to be taking up lots of space when
looking at the page, right on top. Maybe there is a different way to
distinguish between the two sets of badges, without the huge headings.

Please feel free to change the PR and squash it too - I did't realize that
multiple changes I made in the fork will all become part of the PR.

ShreeDevi
____________________________________________________________
भजन - कीर्तन - आरती @ http://bhajans.ramparivar.com

On Thu, Jun 1, 2017 at 2:20 AM, Egor Pugin <notifications@github.com> wrote:

> *@egorpugin* commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/tesseract-ocr/tesseract/pull/964#discussion_r119470371>
> :
>
> > @@ -1,16 +1,9 @@
>  # Tesseract OCR
>
> -For the latest online version of the README.md see:
> -
> -https://github.com/tesseract-ocr/tesseract/blob/master/README.md
> -
> -### Build
> -[![Build Status](https://travis-ci.org/tesseract-ocr/tesseract.svg?branch=master)](https://travis-ci.org/tesseract-ocr/tesseract)
> -[![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819/branch/master?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
> -
> -### Other
>
> Where? The idea behind this caption was to split badges.
> In this PR they are again near each other.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tesseract-ocr/tesseract/pull/964#discussion_r119470371>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE2_o0cXztdqlgCKlXwmpmHI-VcH1EtZks5r_dKugaJpZM4NrqXr>
> .
>

egorpugin commented:
Badges look ugly now.
https://github.com/Shreeshrii/tesseract/blob/master/README.md

Current form is better.

Shreeshrii commented:
@egorpugin I have reverted the change for badges and updated link for 3.05.01 release. Please make any other changes that you see fit.

Also, please add your name as well as those of other maintainers/members with write access to the repo.
